CHANGELOG.md

```diff
@@ -1,5 +1,74 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.4.1
+
+<!-- generated comparing v5.4.0..master -->
+
+_Feb 7, 2022_
```

Comment:
```suggestion
_Feb 8, 2022_
```

Reply:
Now I know why I should complete the release within the day 😂, so that I don't miss the date.